pkg/cluster/provisioner/terraform/cmd_linux.go

```diff
@@ -34,7 +34,13 @@ func (t *terraform) runCmd(action string, args []string, showOutput bool) (int,
 	if ui.Debug() {
 		cmd.Env = append(cmd.Env, "TF_LOG=INFO")
 	}
-
+	if os.Getenv("HTTPS_PROXY") != "" || os.Getenv("https_proxy") != "" {
+		proxyValue := os.Getenv("HTTPS_PROXY")
+		if proxyValue == "" {
+			proxyValue = os.Getenv("https_proxy")
+		}
+		cmd.Env = append(cmd.Env, fmt.Sprintf("HTTPS_PROXY=%s", proxyValue))
```

Comment:
Can we get rid of the lower case option and only use the upper case one (`HTTPS_PROXY`)?

Reply:
done!

Reply:
Thanks :)